### PR TITLE
Steam Test Server - Update 24

### PR DESCRIPTION
**Pull request by Sunaries**
Proofread:
- https://github.com/Treeofsavior/EnglishTranslation/commit/eb150d85bce0bf89f2d6bc2093f28a8ead1f5cdc
- https://github.com/Treeofsavior/EnglishTranslation/commit/b42e8103617986e99b39b28e17c41dbf39d4e3c8

### CODE REVIEW

File: EnglishTranslation-master/ETC.tsv [hunk 1]
```diff
@@ -8104,7 +8104,7 @@ ETC_20150317_008104	Name#Hot Panto Warrior#Level#50
 ETC_20150317_008105	Name#Hot Panto Spearman#Level#50
 ETC_20150317_008106	Wandering Vubbe Archer
 ETC_20150317_008107	with you
-ETC_20150317_008108	Killed. The summon army will be created.
+ETC_20150317_008108	defeated. A summon army will be created.
```
Comment (TentuZero) [inline EnglishTranslation-master/ETC.tsv:8107]: Shouldn't that defeated be capitalized?

**Comment by mamoruk** [comment]
ETC_20160224_020797 Earth Tower 12F——》ETC_20160224_020797 Earth Tower 13F

File: EnglishTranslation-master/ETC.tsv (hunk 1 above)
Comment (Sunaries) [inline EnglishTranslation-master/ETC.tsv:8107]: I made it like that since the Korean line begins with a particle.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Pull request merged**